### ISSUE TITLE
Search completion unactivated by default

### ISSUE THREAD

**Issue by PierreMardon** [brave/brave-browser#7263]
## Description 

With my installation / setup scenario, I was unable to have predictive search working easily.

## Steps to Reproduce

   1. Install Brave
   2. Choose to migrate settings from Chrome (that had a full Google configuration)
   3. **Not sure**: switch search engine in the settings
   4. With Qwant as search engine, try to get some predictive search from the URL bar
   5. As it failed, go to settings and enable the predictive search service
   6. Search again "popular" words like `trump ` => still failing
   7. Go to settings to switch search engine to some other value and back to Qwant again
   8. Yay, now it works!

## Actual result:

No predictive search results even after activating the setting.

## Expected result:
Predictive search result enabled and working: 
- just after install + migrate from Chrome where it's enabled by default (?)
- at least after enabling it in the settings

## Reproduces how often: 

Only once, didn't try again

## Brave version (brave://version info)

```

Brave | 1.0.1 Chromium: 78.0.3904.108 (Build officiel) (64 bits)
-- | --
Révision | 4b26898a39ee037623a72fcfb77279fce0e7d648-refs/branch-heads/3904@{#889}
Système d'exploitation | macOS Version 10.14.6 (assemblage 18G1012)
JavaScript | V8 7.8.279.23
Flash | 32.0.0.293 /Library/Internet Plug-Ins/PepperFlashPlayer/PepperFlashPlayer.plugin
Agent utilisateur | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36
Ligne de commande | /Applications/Brave Browser.app/Contents/MacOS/Brave Browser --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --enable-oop-rasterization=Enabled --sync-url=https://no-thanks.invalid --enable-features=NewExtensionUpdaterService,PasswordImport,WebUIDarkMode,SimplifyHttpsIndicator --disable-features=LookalikeUrlNavigationSuggestionsUI,UnifiedConsent,SyncUSSBookmarks,SmsReceiver,AudioServiceOutOfProcess,NotificationTriggers,AutofillServerCommunication --flag-switches-begin --flag-switches-end
Chemin d'accès exécutable | /Applications/Brave Browser.app/Contents/MacOS/Brave Browser
Chemin d'accès au profil | /Users/pierremardon/Library/Application Support/BraveSoftware/Brave-Browser/Default

```

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yep
- Can you reproduce this issue with the beta channel? Didn't try
- Can you reproduce this issue with the dev channel? Didn't try
- Can you reproduce this issue with the nightly channel? Didn't try

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? Didn't try
- Does the issue resolve itself when disabling Brave Rewards? Didn't try
- Is the issue reproducible on the latest version of Chrome?  Nope

## Miscellaneous Information:

> Sorry for not trying with specific builds or trying to toggle brave specific features. My experience the last few days with brave was so nice that I wanted to report the only small bug to kind of help, and that's it.

Only extra things I installed:
- Dark Reader extension v4.8.5, enabled by default on every website
- Google Windevine :'(

**Comment by rebron** [comment]
Marking wontfix. For privacy reasons, we don't enable autocomplete search suggestions by default.

**Comment by PierreMardon** [comment]
I'm not sure you got my point @rebron :

> 5. As it failed, go to settings and enable the predictive search service
> 6. Search again "popular" words like trump => still failing

This is definitely a bug. I had to switch the selected search engine to make it work properly.
